meson.build

```diff
@@ -363,30 +354,31 @@ if get_option('tests')
                      'test/scale_test.c',
                      'test/tests.c')
 endif
+features += {'tests': get_option('tests')}
 
 includedir = []
 win32_pthreads = get_option('win32-internal-pthreads').require(
     win32 and not posix,
     error_message: 'the os is not win32!',
 )
 if win32_pthreads.allowed()
-    features += 'win32-internal-pthreads'
     flags += ['-isystem', '-I', '-DIN_WINPTHREAD']
```

Comment:
isn't the `-isystem -I` supposed to be right next to the `osdep/win32/include`?
adding it to the end of includedir wouldn't guarantee any of that

Reply:
Well includedir is only used here and these are the last flags added so I think it works. Truth be told, this whole thing is very confusing to me.

Reply:
Well I don't expect you to fix it but I think a comment mentioning the huge hack would be good.